PSICompile/ILCodeGen.cs

```diff
@@ -131,11 +144,16 @@ public class ILCodeGen : Visitor {
          Out ("    call string [System.Runtime]System.String::Concat (string, string)");
       else {
          string op = b.Op.Kind.ToString ().ToLower ();
-         op = op switch { "mod" => "rem", "eq" => "ceq", "lt" => "clt", _ => op };
+         bool cmpeq = (op is "neq" or "geq" or "leq");
+         op = op switch { "mod" => "rem", "eq" or "neq" => "ceq", "lt" or "geq" => "clt", "gt" or "leq" => "cgt", _ => op };

```

Comment:
You could have a multi-line string in the case expression output like:
`"geq" => "clt\n    ldc.i4.0\n    ceq"`
That might make this code cleaner. In principle, IL accepts even multiple op-codes on the same line, so you could even output just `clt  ldc.i4.0  ceq` on one line and it will work. However, that is non-conventional and I would discourage that.

Reply:
updated the code with multi-line string as follows:
![image](https://github.com/tarydon/PSI/assets/125174306/769c7a5c-b934-4e8b-abba-e2b76e5d62e9)